### PR TITLE
Updating doc adding PublicNode

### PR DESCRIPTION
**Pull request by wycks**
Adding PublicNode to Community Providers for C-Chain RPC.

### CODE REVIEW

File: docs/apis/avalanchego/public-api-server.md [hunk 1]
```diff
@@ -247,3 +247,19 @@ Websockets are available for the C-chain and the X-chain.
...
+
+### PublicNode
+
+[PublicNode](https://avalanche.publicnode.com) supports C-Chain
+
+Features:
+
+- Free
+- Globally distributed infrastructure (our own bare metal)
+- Optimized for Speed and reliability - Check our page for stats
```
Comment (yulin-dong) [inline docs/apis/avalanchego/public-api-server.md:259]: Please update to proper letter cases.
```suggestion
- Optimized for speed and reliability - check our page for stats
```

File: docs/apis/avalanchego/public-api-server.md [hunk 2]
```diff
@@ -247,3 +247,19 @@ Websockets are available for the C-chain and the X-chain.
 
 - For C-Chain API, the URL is `https://nd-123-145-789.p2pify.com/API_KEY/ext/bc/C/ws`
 - For X-Chain API, the URL is `https://nd-123-145-789.p2pify.com/API_KEY/ext/bc/X/events`
+
+### PublicNode
+
+[PublicNode](https://avalanche.publicnode.com) supports C-Chain
```
Comment (yulin-dong) [inline docs/apis/avalanchego/public-api-server.md:253]:
```suggestion
[PublicNode](https://avalanche.publicnode.com) supports the C-Chain on the Mainnet.
```

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/public-api-server.md [hunk 3]
```diff
@@ -247,3 +247,19 @@ Websockets are available for the C-chain and the X-chain.
...
+
+Features:
+
+- Free
+- Globally distributed infrastructure (our own bare metal)
```
[the remark this answers is not in the document]
Comment (wycks) [reply docs/apis/avalanchego/public-api-server.md:258]: Yes Allnodes runs it own datacenter for avalanche validators. PublicNode is a free service  run by Allnodes.

[the remark this answers is not in the document]
Comment (yulin-dong) [reply docs/apis/avalanchego/public-api-server.md:258]: `Run by` or `run on` AllNodes? Are you part of AllNodes?

[the remark this answers is not in the document]
Comment (wycks) [reply docs/apis/avalanchego/public-api-server.md:258]: Run by Allnodes on Allnodes Validators, yes I work for Allnodes. But this is a free service and not part of our business or main website.

[the remark this answers is not in the document]
Comment (yulin-dong) [reply docs/apis/avalanchego/public-api-server.md:258]: I would suggest you change it to "Globally distributed infrastructure on Allnodes"

[the remark this answers is not in the document]
Comment (wycks) [reply docs/apis/avalanchego/public-api-server.md:258]: Will do thank your for your guidance

**Comment by cgcardona** [comment]
Closing in favor of https://github.com/ava-labs/avalanche-docs/pull/1170. Thanks!

**Pull request closed without merging**